composer.json

```
@@ -4,7 +4,7 @@
     "require": {
         "php": "^7.4|^8.1",
         "magento/product-community-edition": "^2.4.5",
-        "brick/money": "0.5",
+        "brick/money": "0.7.1",
```

```suggestion
        "brick/money": "0.5.*|0.6.*|0.7.*",
```

we still want to support he old magento